browser/pages/confirm/controller.js

```diff
@@ -35,14 +35,10 @@ class ConfirmController {
     // IF the JDK is not Configured then you can't install devstudio
     $scope.$watch('checkboxModel.cdk.selectedOption', function watchCdkSelectionChange(nVal) {
       if(nVal=='install') {
-        if($scope.checkboxModel.vagrant.selectedOption == 'detected'
-          && !$scope.checkboxModel.vagrant.hasOption('detected')) {
-          $scope.checkboxModel.vagrant.selectedOption = 'install';
-        }
-        if($scope.checkboxModel.virtualbox.selectedOption == 'detected'
-          && !$scope.checkboxModel.virtualbox.hasOption('detected')) {
-          $scope.checkboxModel.virtualbox.selectedOption = 'install';
-        }
+        // if($scope.checkboxModel.virtualbox.selectedOption == 'detected'
+        //   && !$scope.checkboxModel.virtualbox.hasOption('detected')) {
+        //   $scope.checkboxModel.virtualbox.selectedOption = 'install';
+        // }
```

Comment:
you right, I'll better keep it it was for CDK 2 until Hyper-V detection is implemented